recipes/devel/ocaml.yaml

```diff
@@ -0,0 +1,44 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "5.3.0"
+
+checkoutSCM:
+    scm: url
+    url: https://github.com/ocaml/ocaml/archive/refs/tags/${PKG_VERSION}.tar.gz
+    digestSHA256: eb9eab2f21758d3cfb1e78c7f83f0b4dd6302824316aba4abee047a5a4f85029
+    stripComponents: 1
+
+buildTools: [m4, autotools]
+buildVars: [STRIP]
+buildScript: |
+    rsync -a --delete $1/ .
+    mkdir -p .bob
+    pushd .bob
+    ln -s $(which ${STRIP}) strip
+    popd
+    export PATH=${PATH}:$(pwd)/.bob
+
+    ./configure \
+            ${AUTOCONF_BUILD:+--build=${AUTOCONF_BUILD}} \
+            ${AUTOCONF_HOST:+--host=${AUTOCONF_HOST}} \
+            ${AUTOCONF_TARGET:+--target=${AUTOCONF_TARGET}} \
```

Comment:
`AUTOCONF_TARGET` is not part of `buildVars`. I guess you have never built a cross-compiler?